argocd-mixin/alerts/argocd-alerts.yml

```diff
@@ -0,0 +1,31 @@
+groups:
+    - name: ArgoCD                                              
+      rules:
+        - alert: ArgoAppOutOfSync                               
+          expr: argocd_app_info{sync_status="OutOfSync"} == 1   
+          for: 1m                                               
+          labels:                                               
+            severity: warning
+          annotations:                                          
+            summary: '{{ $labels.name }} is OutOfSync' 
+            description: '{{ $labels.name }} Application has sync status as {{ $labels.sync_status }}'
```

Comment:
Linting rules state that the summary should not have templates in it, and that the description should use sentence case (capital first letter, and end in a period).

Reply:
Updated, Thanks!